### PR TITLE
aman's card

### PR DESCRIPTION
**Pull request by Aman7445**

### CODE REVIEW

**Comment by Aman7445** [comment]
Thank you for this begineer friendly project to contribute to for the hacktoberfest . Please merge this PR with the hacktoberfest 2021 tag.Also if you have more projects in which i can provide a good contribution with tech stack of MERN or HTML/CSS and java script please do let me know .Thank you once again :)

File: index.html [hunk 1]
```diff
@@ -5819,6 +5829,52 @@ <h2>A project for first-time contributions</h2>
         </div>
         <!-- ________*CZ's card END ________  -->
 
+        <!-- ________ Aman Contributor card START ________  -->
```
Comment (iatharva) [inline index.html:5832]: Thank you for the contribution, I won't be able to mark it with the tag as we are not in hacktoberfest but still contribution and collaboration is what matters.

Looks like you have added your card at the end of the file, you have to add your card just below the template as per instructions given in step 5, you can check it [here](https://github.com/Syknapse/Contribute-To-This-Project/).
Just do the changes and then we'll review the PR and merge it.

**Comment by Aman7445** [comment]
hey i have made the changes can you see and merge it now.No problem for the hacktoberfest issue ,It was a good learning practice

**Comment by iatharva** [comment]
Looks good. Thanks for the contribution again, and hope you have a great journey ahead in open source. 💯

**Pull request merged**